### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by PrinceBadru**
## Magicians : Full Website.
- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- Add your own styles to improve the look and feel.
- Use React basic styling (inline styles, importing CSS and pre-processed CSS files).
- For every page, you should create a <Route>.
- Make sure that navigation links for all 3 routes are displayed on each page.
- Use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
+import './last/nav.css';
 
 export default function App() {
   return (
-    <div className="App">
-      <Calculator />
-      <Quote />
-    </div>
+    <>
+      <div className="App">
```
Comment (gemmen29) [inline src/App.js:11]: - [ ] Kindly fix this error:
image
When I run your project I get this error.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @PrinceBadru, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bantealem — APPROVED** [review]
# Status: Approved approved

Hi @PrinceBadru, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Screenshot from 2023-09-27 20-56-29


## Highlights
- [x] Correct Git flow
- [x] Professional Readme
- [x] No Linters error
- [x] Descriptive Readme
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — COMMENTED** [review]
# Approved :green_circle: 
Hi @PrinceBadru, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] Application functions as expected :white_check_mark: 
- [x] Git flow is used correctly :white_check_mark:  
- [x] Repository has a professional readme file :white_check_mark: 
- [x] Pull request has a descriptive readme file :white_check_mark:    

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**